doc/building.md

```diff
@@ -422,13 +422,8 @@ To use clang instead of gcc on Linux, use `--with-toolchain-type=clang`.
 
 The oldest supported version of Xcode is 13.0.
 
-You will need the Xcode command line developer tools to be able to build the
-JDK. (Actually, *only* the command line tools are needed, not the IDE.) The
-simplest way to install these is to run:
-
-```
-xcode-select --install
-```
+You will need to download Xcode either from the App Store or specific versions
+can be easily located via the [Xcode Releases](https://xcodereleases.com) website.
```

Comment:
This file is trying to keep a consistent 80 char line length limit and this line is longer than 80.
```suggestion
can be easily located via the [Xcode Releases](https://xcodereleases.com)
website.
```